web/.eslintrc.js

```diff
@@ -0,0 +1,105 @@
+/*
+ reference: https://github.com/toshi-toma/eslint-config-airbnb-typescript-prettier
+*/
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    sourceType: "module",
+    project: "./tsconfig.json",
+    tsconfigRootDir: "./",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  env: {
+    browser: true,
+    es2017: true,
+  },
+  extends: [
+    "airbnb",
+    "airbnb/hooks",
+    "prettier",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/typescript",
+  ],
+  plugins: ["react", "jsx-a11y", "import", "prettier", "@typescript-eslint", "simple-import-sort",],
+  ignorePatterns: [".eslintrc.js", "webpack.config.js"],
+  globals: {},
+  rules: {
+    // prettier
+    "prettier/prettier": ["error"],
+
+    // TypeScript
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/ban-ts-ignore": "off",
+    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/explicit-member-accessibility": "off",
+    "@typescript-eslint/no-object-literal-type-assertion": "off",
+    "@typescript-eslint/no-shadow": "warn",
+    "@typescript-eslint/ban-types": "warn",
+    "@typescript-eslint/no-explicit-any": "warn",
+    "@typescript-eslint/no-shadow": "warn",
+    /* 명시적인 return type 작성*/
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+
+    // js
+    "camelcase": "off",
+    "react/require-default-props": "off",
+    "consistent-return": "off",
+    /* redux toolkit immer js를 위한 세팅 */
+    'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['state'] }],
+    "no-underscore-dangle": "warn",
+    "eqeqeq": "warn",
+    "radix": "warn",
+    "no-plusplus": "warn",
+    "default-case": "warn",
+    "no-restricted-properties": "warn",
+
+    // v4 changes
+    "no-use-before-define": "off",
+    "no-shadow": "off",
+
+    // jsx-a11y
+    "jsx-a11y/click-events-have-key-events": "warn",
+    "jsx-a11y/no-noninteractive-element-interactions": "warn",
+
+    // React
+    "react/jsx-filename-extension": ["error", { extensions: [".tsx"] }],
+    "react/prop-types": "off",
+    "react-hooks/exhaustive-deps": "off",
+    "react/jsx-props-no-spreading": "off",
+    "react/no-array-index-key": "warn",
+    "react/button-has-type": "warn",
+
+    // import
+    "import/no-cycle": "off",
+    "import/prefer-default-export": "off",
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
+        js: "never",
+        mjs: "never",
+        jsx: "never",
+        ts: "never",
+        tsx: "never",
+      },
+    ],
+  },
+
```

Comment:
요거 추가해주면 좋을 것 같습니다 (로컬에서 돌려보니 테스트 코드에 있는 dependencies를 dev로 옮기라고 경고가 떠서)

```suggestion
"import/no-extraneous-dependencies": ["error", {"devDependencies": ["/**/*.ts?(x)"]}
```

Reply:
반영완료
https://github.com/banksalad/styleguide/pull/35/commits/2bbf7c108912d4a95658a930670d4f05db8d667c